modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -712,6 +752,7 @@ private[codegen] class Renderer(compilationUnit: CompilationUnit) { self =>
     )
   )
 
+  // also known as newtypes
```

Comment:
personally I found it a bit confusing that it said "type alias". I could rename the method if it feels like a better name: `renderNewtype`

Reply:
Yeah happy to change the name

Reply:
0bc490d7248790ba975c11dc4b3ea4700bf807f9